Stable release to production/main

devark28 opened the pull request:
## ✅ What
 
Done everything and fixed all the bugs and requested changes
 
## 🤔 Why
 
Staging is now stable and ready for production.
 
## 👩‍🔬 How to validate
 
Everything works as in the design.
 
## 🔖 Further reading
 
- We didn't focus on non-urgent changes like renaming images

MizeroR committed:
Finished the mobile navbar

MizeroR committed:
Finished mobile navbar

MizeroR committed:
Chore: New added images

devark28 committed:
Chore: ignore ide related files

devark28 committed:
UPD: Implement the hero section text and shapes for all breakpoints

Hassanadelani1 committed:
completed the navbar

MEDATTA0 committed:
chore(deps): update tailwindcss

Hassanadelani1 committed:
made the navbar for the tablet

devark28 committed:
FIX: fix the hero section's shapes animations and semantics

UPD: Add sticky animation to the hero shapes
CHORE: Replace the hero's wrapper div with a section

MizeroR committed:
Add yellow shape and the about paragraph

MizeroR committed:
Tablet and desktop version of the section

devark28 committed:
Fix: fix hero text positioning when scrolling on mobile

Hassanadelani1 committed:
finished the navbar section for all breakpoints

devark28 committed:
Fix: fix hero text positioning when scrolling on tablet

MizeroR committed:
A new feature about the Ace Hotel

devark28 committed:
FIX: remove hero shapes dragabbility(pointer) and selectability

Hassanadelani1 committed:
made the acehotel section for the md and lg breakpoints

devark28 committed:
Fix: fix hero shapes positioning on desktop breakpoint

MizeroR committed:
Footer branch

Hassanadelani1 committed:
Chore: removed the .DS_Store files

MEDATTA0 committed:
style(css): make ui for destop breakpoint

MEDATTA0 committed:
Revert "style(css): make ui for destop breakpoint"

This reverts commit d0ae097696ee00a49cb53cb70bdefd1443d7a9f7.

Hassanadelani1 committed:
Chore: Update the tailwind config file to include the root HTML files

MEDATTA0 committed:
style(css): finish the footer for all breakpoint

sjamillah committed:
feature: the brand strategy section content

devark28 committed:
Merge pull request #4 from Group-3-Website-Project/ft/hero

UPD: Implement the hero section text and shapes for all breakpoints

devark28 committed:
Merge pull request #6 from Group-3-Website-Project/ft/issue#5

Fix: fix tailwind dependencies vulnerability issue

devark28 committed:
Merge pull request #2 from Group-3-Website-Project/ft/navbar

UPD: Implement the nav bar

devark28 committed:
Merge branch 'dev' into ft/acehotel

devark28 committed:
Merge pull request #8 from Group-3-Website-Project/ft/acehotel

Update: build acehotel Section

devark28 committed:
Chore: exclude mac os' Data Store files

devark28 committed:
Fix: hero text escapes hero section stacking context (issue#12)

devark28 committed:
Merge branch 'dev' into ft/brand-strategy

Hassan-Adelani-Luqman committed:
Fix: unelevated nav and nav items

MizeroR committed:
Change of the paragraphs order

Hassan-Adelani-Luqman committed:
Chore: Added the contact.html file

Hassan-Adelani-Luqman committed:
Revert "Chore: Added the contact.html file"

This reverts commit 0716cca0cd66d1dcc3dee96c8a1614bedce9d949.

devark28 committed:
Merge pull request #13 from Group-3-Website-Project/bg/bleeding_hero_text

Fix: hero text escapes hero section stacking context (issue#12)

Hassan-Adelani-Luqman committed:
Chore: Added the contact.html file

Hassan-Adelani-Luqman committed:
Chore: Added the imprint.html file

sjamillah committed:
fix: made the green shape in the medium screens aligned well

MizeroR committed:
Merge pull request #16 from Group-3-Website-Project/bg/issue#11

Fix: unelevated nav and nav items

sjamillah committed:
fix: added padding bottom and the grid rows to reduce the extra space

devark28 committed:
Merge pull request #9 from Group-3-Website-Project/ft/brand-strategy

Made the brand strategy section

MizeroR committed:
Main section in the contact page

MizeroR committed:
Merge pull request #10 from Group-3-Website-Project/ft/footer

Ft/footer

devark28 committed:
Fix: sticky navbar issue#15

Hassan-Adelani-Luqman committed:
Fix: missing the dropdown background shapes

MizeroR committed:
Merge branch 'dev' into ft/imprint

devark28 committed:
Fix: scrollable nav dropdown (issue#21)

MEDATTA0 committed:
feat(): create the data privacy policy page

Hassan-Adelani-Luqman committed:
Fix: Fix issue 14

Hassanadelani1 committed:
Merge branch 'dev' into bg/issue#14

Hassan-Adelani-Luqman committed:
Fix: fix the desktop version of the navbar

Hassan-Adelani-Luqman committed:
Fix: Remove the contact section from the landing page

MizeroR committed:
Merge pull request #23 from Group-3-Website-Project/bg/scrollable_dropdown

Fix: scrollable nav dropdown (issue#21)

Hassan-Adelani-Luqman committed:
Fix: Change the size of the logo on the navbar to adapt to the desktop version

MizeroR committed:
Merge pull request #18 from Group-3-Website-Project/ft/imprint

Ft/imprint

MizeroR committed:
Merge branch 'dev' into bg/static_nav

MizeroR committed:
Merge pull request #19 from Group-3-Website-Project/bg/static_nav

Fix: sticky navbar (issue#15)

sjamillah committed:
fix: made the yellow shape sticky

devark28 committed:
Merge branch 'dev' into bg/issue#14

sjamillah committed:
fix: adjusted the line height of the text

MizeroR committed:
Merge pull request #27 from Group-3-Website-Project/bg/sticky-yellow-shape

fix: made the yellow shape sticky

MizeroR committed:
Merge pull request #25 from Group-3-Website-Project/bg/contactSectionRemoval

Fix: Remove the contact section from the landing page

MizeroR committed:
Merge pull request #24 from Group-3-Website-Project/bg/desktopNavbar

Fix: fix the desktop version of the navbar

MizeroR committed:
Fix: Changed logo to text

MizeroR committed:
Chore: Deleted image

MizeroR committed:
Fix: Add a background shape in nav on mobile

MizeroR committed:
Fix: Change the color of the burger menu

MizeroR committed:
Fix: Link to contact page

MizeroR committed:
Fix: Make baumesteir go to index

MizeroR committed:
Fix: Delete href to imprint

MizeroR committed:
Merge pull request #28 from Group-3-Website-Project/bg/logo

Fix: Changed logo from image to text

MEDATTA0 committed:
fix(css): logo background behavior

sjamillah committed:
fix: renamed the wrappers of the sections and deleted the contact.html file

devark28 committed:
Merge branch 'dev' into bg/issue#14

MizeroR committed:
Merge pull request #33 from Group-3-Website-Project/31-logo-background-shape

fix(css): logo background behavior

MizeroR committed:
refactor: modified empty alt attributes and changed titles

sjamillah committed:
chore: made the privacy policy page responsive and added styles

MizeroR committed:
Merge branch 'dev' into renaming-the-file-wrappers

MizeroR committed:
Merge pull request #34 from Group-3-Website-Project/renaming-the-file-wrappers

fix: renamed the wrappers of the sections and deleted the contact.htm…

sjamillah committed:
fix: added padding for all the screens

MizeroR committed:
Merge branch 'dev' into bg/issue#14

MizeroR committed:
Merge pull request #22 from Group-3-Website-Project/bg/issue#14

Fix: missing the dropdown background shapes (issue#14)

sjamillah committed:
fix: fixing the padding along the x-axis of the acehotel on all breakpoints

MizeroR committed:
Fix: Properly linking legal notice file to imprint page

MizeroR committed:
Merge pull request #38 from Group-3-Website-Project/bg/imprint

Fix: Properly linking legal notice file to imprint page

MizeroR committed:
refactor: Editing file titles

MizeroR committed:
Merge branch 'bg/alt-attributes' into dev

MizeroR committed:
Merge pull request #36 from Group-3-Website-Project/ft/data-privacy

Ft/data privacy

MizeroR committed:
fix: vertical padding

MizeroR committed:
Merge branch 'dev' into bg/ace-hotel-padding

MizeroR committed:
Merge pull request #37 from Group-3-Website-Project/bg/ace-hotel-padding

fix: fixing the padding along the x-axis of the acehotel on all break…

devark28 committed:
Chore: fix ul wrapping links instead of li elements

devark28 committed:
Chore: fix a wrapping block element

MizeroR committed:
Chore: Proper path of data store

MizeroR committed:
Merge pull request #42 from Group-3-Website-Project/chore/data-store

Chore: Proper path of data store

devark28 committed:
Chore: fix privacy policy page and imprint page titles

Hassan-Adelani-Luqman committed:
Feat: Add imprint and privacy policy links to footer

Hassan-Adelani-Luqman committed:
Chore: replaced the some tags in the footer section with semantic tags

Hassan-Adelani-Luqman committed:
Chore: moved misplaced classes

devark28 committed:
Merge pull request #43 from Group-3-Website-Project/bg/page-titles

Chore: fix privacy policy page and imprint page titles

devark28 committed:
Merge pull request #41 from Group-3-Website-Project/39-li-wrapping-a-not-the-other-way

39 li wrapping a not the other way

MizeroR committed:
Merge pull request #44 from Group-3-Website-Project/ft/footer-links

Feat: Add imprint and privacy policy links to footer

devark28 committed:
Merge pull request #29 from Group-3-Website-Project/dev

Merge to staging branch

devark28 committed:
Fix: fix big menu button on mobile issue

devark28 committed:
Merge branch 'dev' into 30-large-burger-menu

KevinRebakure commented on src/sections/brand-strategy.html:
```diff
@@ -0,0 +1,61 @@
+<div
+  class="w-full p-10 sm:p-10 md:p-20 lg:pb-64 grid-cols-1 grid-rows-2 md:grid-cols-1 lg:grid-cols-2 lg:grid-rows-1 lg:gap-20 lg:px-36 md:grid flex flex-col"
+>
+  <div
+    class="text-black font-normal space-y-4 lg:text-[40px] md:space-y-12 lg:space-y-24"
+  >
+    <h2 class="text-xl sm:text-xl md:text-[35px] lg:text-[48px]">
+      Brand strategy
+    </h2>
+    <ul class="list-none space-y-4 md:space-y-12 lg:space-y-24 text-xl sm:text-xl md:text-[35px] lg:text-[48px] break-words md:leading-[1.2] lg:leading-[1.2]">
```
Wasn't expecting to see these arbitrary values again.

KevinRebakure commented on src/sections/header.html:
```diff
@@ -1,4 +1,47 @@
+            <p>EN</p>
+        </div>
+
+        <a class="flex items-center justify-center" href="index.html">
+            <div class="md:hidden">
+                <img src="/assets/img/gray.svg" style="display: var(--logo-bg, inline-block);" alt="Gray Background Shape"
+                    class="absolute -translate-x-3 -translate-y-5 -z-10">
+            </div>
+            <span class="text-3xl font-semibold md:-mt-8 md:text-5xl">Baumeister</span>
+        </a>
```
An `a` wrapping a `div` element???

KevinRebakure reviewed (commented):
Finalize the last 2 remaining issues and merge it.

MEDATTA0 committed:
fix(ui): nav dropdown not showing on non-index pages

devark28 committed:
Chore: remove more arbitrary values

devark28 committed:
Chore: fix a wrapping a div semantic

MizeroR committed:
Fix: privacy-policy html injector not returning a promise

devark28 committed:
Merge pull request #51 from Group-3-Website-Project/chore/49-a-wrapping-a-div-element

Chore: fix a wrapping a div semantic

devark28 committed:
Merge pull request #46 from Group-3-Website-Project/30-large-burger-menu

Fix: fix big menu button on mobile issue

devark28 committed:
Fix: make nav dropdown fixed

Made the nav dropdown fixed so that when you scroll and open the nav you can still see the dropdown.

devark28 committed:
Merge pull request #47 from Group-3-Website-Project/ft/issue#5

fix(ui): nav dropdown not showing on non-index pages

devark28 committed:
Fix: non fixed nav dropdown

devark28 committed:
Merge pull request #52 from Group-3-Website-Project/bg/non-fixed-nav-dropdown

Fix: non fixed nav dropdown

MizeroR committed:
Merge pull request #50 from Group-3-Website-Project/bg/48-remove-more-albitrary-values

Chore: remove more arbitrary values (check if it all still work!!!!!)

MizeroR committed:
Fix: make imprint page navbar sticky

MizeroR committed:
Fix: make privacy-policy page navbar sticky

MizeroR committed:
fix: Header section background color and logo background shape

MizeroR committed:
Fix: Header section background color and logo background shape

MizeroR committed:
Fix: Header section

MizeroR committed:
fix: logo bg shape positioning

MizeroR committed:
Merge pull request #55 from Group-3-Website-Project/bg/54-navbar-isnt-sticky-for-non-index-pages

Bg/54 navbar isnt sticky for non index pages

devark28 committed:
Chore: fix data privacy _ and - naming inconcistencies

Hassan-Adelani-Luqman committed:
Fix: Corrected inline element wrapping a block element

MizeroR committed:
Fix: Links to the sections in nav dropdown

devark28 committed:
Merge pull request #60 from Group-3-Website-Project/fix/issue#56

Fix: Corrected inline element wrapping a block element

devark28 committed:
Merge pull request #59 from Group-3-Website-Project/bg/58-data_privacyhtml-with-data-privacy-inconsistency

Chore: fix data privacy _ and - naming inconcistencies

devark28 committed:
Merge pull request #61 from Group-3-Website-Project/bg/57-nav-dropdown-links-dont-go-anywhere

Fix: Links to the sections in nav dropdown

devark28 committed:
Merge pull request #53 from Group-3-Website-Project/dev

Refactor Header and Menu Functionality, Improve Responsiveness, and Update Styling Across Multiple Pages, Fix broken about section navigation.
